include/pyjs/pre_js/load_pkg.js

```diff
@@ -285,6 +286,7 @@ Module["bootstrap_from_empack_packed_environment"] = async function
         if(verbose){
             console.log("fetchAndUntarAll");
         }
+        await Promise.all(all_mount_points.map(pkg => fetchAndUntar(package_tarballs_root_url, python_is_ready_promise, pkg, verbose)));
```

Comment:
atm this will wait for **all** mounts to be fetchedAndUntared even though we could already start with the download of the packages. 
Might make sense to modify this st. all awaitables are awaited in one `await Promise.all`